tests/forms_tests/field_tests/test_genericipaddressfield.py

```diff
@@ -5,9 +5,15 @@
 class GenericIPAddressFieldTest(SimpleTestCase):
 
     def test_generic_ipaddress_invalid_arguments(self):
-        with self.assertRaises(ValueError):
+        with self.assertRaises(ValueError) as context:
             GenericIPAddressField(protocol='hamster')
-        with self.assertRaises(ValueError):
+        exception_msg = str(context.exception)
+        self.assertIn("The protocol 'hamster' is unknown. Supported: ", exception_msg)
+        for protocol in ['ipv6', 'ipv4', 'both']:
```

Comment:
If the issue here is non-deterministic sorting, you could adjust the validator in a separate PR.